### PR TITLE
replacing screenshots for ET user hierarchy

### PR DESCRIPTION
**Pull request by darluk**
screenshots update for https://cumulocity.com/guides/10.16.0/users-guide/enterprise-tenant/#user-hierarchies

### CODE REVIEW

**Comment by BeateRixen** [comment]
@darluk I would actually recommend to remove the following screenshots as they are not really required:
et-user-hierarchy-delegate
et-user-hierarchy-roles

**Review by BeateRixen — APPROVED** [review]

**Comment by darluk** [comment]
> @darluk I would actually recommend to remove the following screenshots as they are not really required: et-user-hierarchy-delegate et-user-hierarchy-roles

I would leave both because in UI it's not really very clear where it can be seen with a new layout

**Comment by BeateRixen** [comment]
> > @darluk I would actually recommend to remove the following screenshots as they are not really required: et-user-hierarchy-delegate et-user-hierarchy-roles
> 
> I would leave both because in UI it's not really very clear where it can be seen with a new layout

Ok, your decision :-).

**Review by jkot-softwareag — APPROVED** [review]

**Comment by darluk** [comment]
> > > @darluk I would actually recommend to remove the following screenshots as they are not really required: et-user-hierarchy-delegate et-user-hierarchy-roles
> > 
> > 
> > I would leave both because in UI it's not really very clear where it can be seen with a new layout
> 
> Ok, your decision :-).

consulted with the team and decided to remove screenshots as @BeateRixen suggested. https://github.com/SoftwareAG/c8y-docs/pull/832/commits/5a21bd204ed5d680de145d878dd9b7724aa00d42

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 1]
```diff
@@ -54,8 +52,6 @@ You may of course also delegate on a temporary basis, for example if you are tem
 
 Either open the user and click the delegate icon in the **Delegated by** field, or click the menu icon at the right of the user entry in the user list and from the context menu, select **Delegate**.
 
-![User delegation](/images/users-guide/enterprise-tenant/et-user-hierarchy-delegate.png)
```
Comment (jkot-softwareag) [inline content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:57]: :+1: this field is visible on the above screenshot with user owner

**Review by jkot-softwareag — APPROVED** [review]

File: content/users-guide/enterprise-tenant-bundle/user-hierarchies.md [hunk 2]
```diff
@@ -39,9 +39,7 @@ If you want an owner to manage only their sub-users, make sure that the owner do
 
 **Example**
 
-A user A has the role "business". User A becomes the owner of a new user B. User  B can then only get a business role assigned (and not for example an admin role) as the user cannot have higher permissions than the owner.
-
-![Owner Sample](/images/users-guide/enterprise-tenant/et-user-hierarchy-roles.png)
+A user A has the role "business". User A becomes the owner of a new user B. User B can then only get a business role assigned (and not for example an admin role) as the user cannot have higher permissions than the owner. While trying to assign any other role except "business" for user B will be shown an exclamation icon for that role with a notification, that only the global role defined for the owner can be defined for owned user.
```
Comment (darluk) [inline content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:42]: also could be changed to ```A user A has the role "business". User A becomes the owner of a new user B. User B can then only get a business role assigned (and not for example an admin role) as the user cannot have higher permissions than the owner. If you try to assign any other role except "business" for user B, then the role will be unavailable for subscription and will be indicated by a warning icon with a notification that this operation is not permitted.``` @BeateRixen what do you think?

Comment (BeateRixen) [reply content/users-guide/enterprise-tenant-bundle/user-hierarchies.md:42]: yes, sounds much better

**Pull request merged**